tekton/images/hub/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM alpine:3.10
+LABEL maintainer "Tekton Authors <tekton-dev@googlegroups.com>"
+
+ARG HUB_VERSION=2.13.0
+RUN wget -O- https://github.com/github/hub/releases/download/v${HUB_VERSION}/hub-linux-amd64-${HUB_VERSION}.tgz | tar zxf - ; mv hub-linux-amd64-${HUB_VERSION}/bin/hub /usr/local/bin; rm -rf hub-linux-amd64-${HUB_VERSION}
```

Comment:
what about something like this ? :

https://github.com/tektoncd/plumbing/pull/129/files#diff-5a3cf5117f7aca60fc7c678ee789c55fR48

Reply:
Oh, nice. Since we build the image nightly though, I think it's not too bad if we use a pinned version, so we can avoid suddenly breaking our automation on a new version.
The downside of course is that it takes a pull request to update the version.

Reply:
It still takes a pull request to pickup the latest version tho, it only grabs the latest one every time we doing a rebuild instead of having to track the version down.

Reply:
Uhm - https://github.com/github/hub/issues/1818
It turns out that pre-built binaries are not statically linked which is not good for alpine.
There is an alpine test repo with hub, but it's probably better to build `hub` directly then.

Reply:
> It still takes a pull request to pickup the latest version tho, it only grabs the latest one every time we doing a rebuild instead of having to track the version down.

We build the images nightly, so, if I understood it right, it would update automatically

Reply:
okay i missed the nightly part, no worries then,